provider/cmd/pulumi-resource-nuage/nuage_provider/serverless_database.py

```diff
@@ -136,8 +148,8 @@ def __init__(
         # Create a cluster instance
 
         aws.rds.ClusterInstance(
-            resource_name=f"{args.resource_name}-cluster-instance",
-            identifier_prefix=f"{name}-instance-",
+            resource_name=resource_name,
+            identifier=self.get_suffixed_name("instance"),
```

Comment:
Same as "cluster" above : we should just name it according to the user's parameters